History.txt

```diff
@@ -1,4 +1,4 @@
-0.9, 2017-08-??
+0.9, 2017-08-14
```

Comment:
IMHO there are too much activity and not enough tests of a "stable" version for releasing today.